server/controllers/documents/upload/helpers/sendFileSizeMetric.test.ts

```diff
@@ -39,13 +39,13 @@ describe('sendFileSizeMetric', () => {
     expect(observe).toHaveBeenCalledWith(10.5)
   })
 
-  it('does not send the metric in development', () => {
+  it('sends the metric in development', () => {
```

Comment:
I'd argue against ever doing anything different per env (for metrics etc) - there's no cost to doing it